Interface/AddOns/oUF_Neav/config.lua

```diff
@@ -135,9 +135,16 @@ ns.Config = {
 
             numBuffs = 20,
             numDebuffs = 20,
+
+            disableAura = false,                                                            -- Disable Auras on this unitframe
             colorPlayerDebuffsOnly = true,
             showAllTimers = false,                                                          -- If false, only the player debuffs have timer
-            disableAura = false,                                                            -- Disable Auras on this unitframe
+            onlyShowPlayer = false,                                                          -- Only shows player created buffs/debuffs.
+
+            -- Debuffs on top options. Overrides onlyShowPlayer option.
+            showDebuffsOnTop = false,
+            onlyShowPlayerDebuffs = true,
+            onlyShowPlayerBuffs = false,
```

Comment:
I made it that way because the odds that you want to see all the buffs on a target but only your debuffs is high.  It may be a tad confusing to new people but I think that people willing to use a oUF layout can handle it.